Realm/RLMObjectBase.mm

```diff
@@ -289,3 +269,31 @@ - (NSUInteger)hash {
 }
 
 @end
+
+
+
+Class RLMObjectUtilClass(bool isSwift) {
+    static Class objectUtilObjc = [RLMObjectUtil class];
+    static Class objectUtilSwift = NSClassFromString(@"RealmSwift.ObjectUtil");
+    return isSwift && objectUtilSwift ? objectUtilSwift : objectUtilObjc;
```

Comment:
We don't support mixing the apis. ie the old apis are not supported with the RealmSwift framework.